### PR TITLE
MMT-1009: Adds ability to delete variable collection associations

### PR DESCRIPTION
**Pull request by abbottry**

### CODE REVIEW

File: app/controllers/collection_associations_controller.rb [hunk 1]
```diff
@@ -57,4 +79,18 @@ def add_high_level_breadcrumbs
 
     add_breadcrumb 'Collection Associations', variable_collection_associations_path(params[:variable_id])
   end
+
+  def log_issues(response)
+    response.body.select { |assocation_response| assocation_response.fetch('warnings', []).any? }.each do |warnings|
+      warnings.each do |warning|
+        Rails.logger.error "Variable Assocation [warning]: #{warning}"
```
Comment (charleshuang80) [inline app/controllers/collection_associations_controller.rb]: I don't know what the warnings or errors look like, but does this provide sufficient information (i.e. the variable and/or collection concept ids)?

File: spec/support/draft_helpers.rb [hunk 2]
```diff
@@ -101,6 +101,16 @@ def publish_variable_draft(provider_id: 'MMT_2', native_id: nil, name: nil, long
       end
     end
 
+    def add_variable_collection_associations(variable_id, collection_concept_ids)
```
Comment (charleshuang80) [inline spec/support/draft_helpers.rb]: I added a similar helper for this in MMT-1124 (and it just got merged). I'm not sure if this PR is the right place to consolidate the two or if we should do that in a future PR.

File: app/controllers/collection_associations_controller.rb (hunk 1 above)
Comment (abbottry) [reply app/controllers/collection_associations_controller.rb]: Yes, it does.

File: spec/support/draft_helpers.rb (hunk 2 above)
Comment (abbottry) [reply spec/support/draft_helpers.rb]: Sure, I'll take a look

**Pull request merged**